references/documents.md

```diff
@@ -92,27 +86,26 @@ $ curl \
 
 <RouteHighlighter method="POST" route="/indexes/:uid/documents"/>
 
-Insert a list of documents or replace them if they already exist based on [their unique identifiers](/guides/main_concepts/indexes.md#schema-definition).
+Add a list of documents or replace them if they already exist.
+
+If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **the whole old document**  will be override by the new document. Thus, removing any fields previously in the document not present in the new document.
```

Comment:
```suggestion
If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **the whole old document** will be overridden by the new document. Thus, removing any fields previously in the document not present in the new document.
```

Reply:
changed with `overwritten`

Reply:
Changed with `If you send an already existing document (same [id](/guides/main_concepts/documents.md#identifier)) the **whole existing document** will be overwritten by the new document. Fields previously in the document not present in the new document are removed.`